src/renderer/views/Settings/Settings.vue

```diff
@@ -22,6 +22,10 @@
         class="settingsContent"
       >
         <div class="switchRow">
+          <ft-button
+            :label="$t('Settings.Show Keyboard Shortcuts')"
+            @click="showKeyboardShortcutPrompt"
+          />
```

Comment:
Fair call out, I didn't know what exactly to do with this one either. This was another adjustment I tried out. Open to this one or any other suggestions. @absidue 

![Screenshot_20241208_085711](https://github.com/user-attachments/assets/16105edf-bbe7-4a70-a3d5-af0d77aecd6f)

Reply:
Maybe it could be under general setting with a keyboard icon added to the button? Or maybe a separate keyboard shortcuts "settings section" (so it wouldnt be using a modal popup and would just display all the shortcuts under the Keyboard Shortcuts section but we'd be able to update it to support https://github.com/FreeTubeApp/FreeTube/issues/251 eventually).

Reply:
If it's its own setting section, I do worry that people will think it's already modifiable. I can put it on its own row there as a button for now if that works, but it might still look out of place and not fit in as a "general setting".